src/Store/VimStoreConnector.re

```diff
@@ -214,6 +214,12 @@ let start =
       dispatch(Actions.BufferSetModified(id, modified));
     });
 
+  let _: unit => unit =
+    Vim.Buffer.onWrite(id => {
+      Log.debugf(m => m("Buffer written: %n", id));
+      dispatch(Actions.BufferSaved(id));
```

Comment:
The log here seems superfluous, when `dispatch` is going to log the action anyways.